{"8f129459ef0776a0fa46da5fc475ea3a":{"statistic":{"failed":17,"broken":1,"skipped":0,"passed":18,"unknown":0,"total":36},"items":[{"uid":"85bff142e619a5b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/85bff142e619a5b","status":"failed","statusDetails":"AssertionError: No successful 'transferADA' log found | Failed Logs: None","time":{"start":1727169153192,"stop":1727169153421,"duration":229}},{"uid":"883fda673b9e0765","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/883fda673b9e0765","status":"failed","statusDetails":"AssertionError: No successful 'transferADA' log found | Failed Logs: None","time":{"start":1727152501866,"stop":1727152502078,"duration":212}},{"uid":"20cf743b5dd548d6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/20cf743b5dd548d6","status":"failed","statusDetails":"AssertionError: No successful 'transferADA' log found | Failed Logs: None","time":{"start":1727148732500,"stop":1727148732703,"duration":203}},{"uid":"41996c917422888f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/41996c917422888f","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/trigger-history?agent_id=cbccb0ca-8534-47ca-95e0-0020248e9820&functionName=transferADA (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088033249,"stop":1727088033477,"duration":228}},{"uid":"f999a5bee35e9dff","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/f999a5bee35e9dff","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571620,"stop":1726811571620,"duration":0}},{"uid":"cf3dc104a4351f3e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/cf3dc104a4351f3e","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987758,"stop":1726738987758,"duration":0}},{"uid":"8e301d229f39cec7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/8e301d229f39cec7","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682611,"stop":1726730682611,"duration":0}},{"uid":"63f0b4a950ee6433","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/63f0b4a950ee6433","status":"passed","time":{"start":1726730663452,"stop":1726730663712,"duration":260}},{"uid":"3ef4a4a4c492f0eb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/3ef4a4a4c492f0eb","status":"passed","time":{"start":1726655637829,"stop":1726655638050,"duration":221}},{"uid":"df01e05ab55daf0b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/df01e05ab55daf0b","status":"passed","time":{"start":1726572393691,"stop":1726572394006,"duration":315}},{"uid":"33fe24ae837bfa1f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/33fe24ae837bfa1f","status":"passed","time":{"start":1726571193184,"stop":1726571193459,"duration":275}},{"uid":"596dc841305e57b4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/596dc841305e57b4","status":"passed","time":{"start":1726555652881,"stop":1726555653177,"duration":296}},{"uid":"1525d9ba890b0cb6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/1525d9ba890b0cb6","status":"passed","time":{"start":1726554683559,"stop":1726554683850,"duration":291}},{"uid":"3a1b6e4c05af7d53","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/3a1b6e4c05af7d53","status":"passed","time":{"start":1726552149497,"stop":1726552149763,"duration":266}},{"uid":"37c3eb3b1df607fe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/37c3eb3b1df607fe","status":"passed","time":{"start":1726475579618,"stop":1726475579937,"duration":319}},{"uid":"5ba99f2fb408df1c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/5ba99f2fb408df1c","status":"passed","time":{"start":1726461465539,"stop":1726461465821,"duration":282}},{"uid":"4d4a696a6ab6d230","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/4d4a696a6ab6d230","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110345,"stop":1726056110345,"duration":0}},{"uid":"3a5f24dc3e3020c8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/3a5f24dc3e3020c8","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674592,"stop":1726051674592,"duration":0}},{"uid":"e3fe69c3cad9c939","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/e3fe69c3cad9c939","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884389,"stop":1726047884389,"duration":0}},{"uid":"4f6fd80d80ad196a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/4f6fd80d80ad196a","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915637,"stop":1726037915637,"duration":0}}]},"011c6a73c3f72f6a7ca823b0704e81e3":{"statistic":{"failed":18,"broken":2,"skipped":0,"passed":16,"unknown":0,"total":36},"items":[{"uid":"94bae2223abdd1c4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/94bae2223abdd1c4","status":"failed","statusDetails":"AssertionError: No successful 'create infoAction proposal' log found | Failed Logs: None","time":{"start":1727169153454,"stop":1727169153676,"duration":222}},{"uid":"5887993078708ee6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/5887993078708ee6","status":"failed","statusDetails":"AssertionError: No successful 'create infoAction proposal' log found | Failed Logs: None","time":{"start":1727152502111,"stop":1727152502317,"duration":206}},{"uid":"872a864229f96119","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/872a864229f96119","status":"failed","statusDetails":"AssertionError: No successful 'create infoAction proposal' log found | Failed Logs: None","time":{"start":1727148732737,"stop":1727148732937,"duration":200}},{"uid":"83972e8fe46deb8e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/83972e8fe46deb8e","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/trigger-history?agent_id=cbccb0ca-8534-47ca-95e0-0020248e9820&functionName=createInfoGovAction (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088033604,"stop":1727088033831,"duration":227}},{"uid":"4e7dc8b70164e8db","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/4e7dc8b70164e8db","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571632,"stop":1726811571632,"duration":0}},{"uid":"9a493af09bde6a62","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/9a493af09bde6a62","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987770,"stop":1726738987770,"duration":0}},{"uid":"16349baffa7a7d86","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/16349baffa7a7d86","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682623,"stop":1726730682623,"duration":0}},{"uid":"85b62939b4caf643","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/85b62939b4caf643","status":"passed","time":{"start":1726730663713,"stop":1726730663930,"duration":217}},{"uid":"e8808b1c877221e6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/e8808b1c877221e6","status":"passed","time":{"start":1726655638051,"stop":1726655638282,"duration":231}},{"uid":"993c1d2f1ff6328d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/993c1d2f1ff6328d","status":"passed","time":{"start":1726572394008,"stop":1726572394312,"duration":304}},{"uid":"7a1bf7f1eae54259","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/7a1bf7f1eae54259","status":"passed","time":{"start":1726571193461,"stop":1726571193680,"duration":219}},{"uid":"b5fbcba8bca04e1b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/b5fbcba8bca04e1b","status":"failed","statusDetails":"AssertionError: No successful 'create infoAction proposal' log found | Failed Logs: [{'agentId': '4c382513-8e75-40c7-acd3-80b9196abaf0', 'functionName': 'createInfoGovAction', 'status': True, 'success': False, 'timestamp': '2024-09-17T06:42:42.525000+00:00', 'message': 'Timeout on RPC call', 'triggerType': 'MANUAL', 'txHash': None, 'instanceIndex': 0}]","time":{"start":1726555653178,"stop":1726555653433,"duration":255}},{"uid":"720ad678db9d4374","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/720ad678db9d4374","status":"failed","statusDetails":"AssertionError: No successful 'create infoAction proposal' log found | Failed Logs: [{'agentId': 'ce72cfec-b76f-446f-a81e-ae75fe654174', 'functionName': 'createInfoGovAction', 'status': True, 'success': False, 'timestamp': '2024-09-17T06:26:33.082000+00:00', 'message': 'Timeout on RPC call', 'triggerType': 'MANUAL', 'txHash': None, 'instanceIndex': 0}]","time":{"start":1726554683851,"stop":1726554684139,"duration":288}},{"uid":"2e457abe7a8585eb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/2e457abe7a8585eb","status":"passed","time":{"start":1726552149764,"stop":1726552150042,"duration":278}},{"uid":"5b7d857cc62ae090","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/5b7d857cc62ae090","status":"passed","time":{"start":1726475579938,"stop":1726475580236,"duration":298}},{"uid":"c72c7186c55ac1d9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/c72c7186c55ac1d9","status":"passed","time":{"start":1726461465822,"stop":1726461466087,"duration":265}},{"uid":"ed575bbe921f0297","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/ed575bbe921f0297","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110357,"stop":1726056110357,"duration":0}},{"uid":"9acdda8b3be9148d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/9acdda8b3be9148d","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674604,"stop":1726051674604,"duration":0}},{"uid":"c945d4157920fe76","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/c945d4157920fe76","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884402,"stop":1726047884402,"duration":0}},{"uid":"63ec8376b3b383de","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/63ec8376b3b383de","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915649,"stop":1726037915649,"duration":0}}]},"4b813c3575320ac912016173b78fbde9":{"statistic":{"failed":10,"broken":0,"skipped":0,"passed":26,"unknown":0,"total":36},"items":[{"uid":"dc8735fb69e0bb29","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/dc8735fb69e0bb29","status":"passed","time":{"start":1727169154668,"stop":1727169154668,"duration":0}},{"uid":"fff31f55b52d34c9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/fff31f55b52d34c9","status":"passed","time":{"start":1727152503281,"stop":1727152503281,"duration":0}},{"uid":"a50d50cd027a3a8e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/a50d50cd027a3a8e","status":"passed","time":{"start":1727148733873,"stop":1727148733873,"duration":0}},{"uid":"d1bbe56bda652f74","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d1bbe56bda652f74","status":"failed","statusDetails":"AssertionError: Expected DELETE /agents/cbccb0ca-8534-47ca-95e0-0020248e9820 to succeed but got statusCode: 502 : body: <html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1727088038883,"stop":1727088038883,"duration":0}},{"uid":"858faeb055740d45","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/858faeb055740d45","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811572209,"stop":1726811572209,"duration":0}},{"uid":"b868463d841dd63d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/b868463d841dd63d","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738988350,"stop":1726738988350,"duration":0}},{"uid":"47492c41745fdfbc","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/47492c41745fdfbc","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730683455,"stop":1726730683455,"duration":0}},{"uid":"bd51ad7fd6f82fae","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/bd51ad7fd6f82fae","status":"passed","time":{"start":1726730664958,"stop":1726730664958,"duration":0}},{"uid":"36bd26e65dd5492e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/36bd26e65dd5492e","status":"passed","time":{"start":1726655639370,"stop":1726655639370,"duration":0}},{"uid":"df8a36d127ba19c3","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/df8a36d127ba19c3","status":"passed","time":{"start":1726572395764,"stop":1726572395764,"duration":0}},{"uid":"24324d7450256c47","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/24324d7450256c47","status":"passed","time":{"start":1726571194851,"stop":1726571194851,"duration":0}},{"uid":"140a65e870cc2a48","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/140a65e870cc2a48","status":"passed","time":{"start":1726555654582,"stop":1726555654582,"duration":0}},{"uid":"a4eeded29bced488","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/a4eeded29bced488","status":"passed","time":{"start":1726554685595,"stop":1726554685595,"duration":0}},{"uid":"8a37045da40be87c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/8a37045da40be87c","status":"passed","time":{"start":1726552151447,"stop":1726552151448,"duration":1}},{"uid":"4faebdbcf7370820","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/4faebdbcf7370820","status":"passed","time":{"start":1726475581679,"stop":1726475581679,"duration":0}},{"uid":"3841bfc8679d0cc0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/3841bfc8679d0cc0","status":"passed","time":{"start":1726461467152,"stop":1726461467152,"duration":0}},{"uid":"6a484d2d7c81aa07","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/6a484d2d7c81aa07","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056111100,"stop":1726056111100,"duration":0}},{"uid":"1ccd0808d43c4cb7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/1ccd0808d43c4cb7","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051675331,"stop":1726051675331,"duration":0}},{"uid":"a653f4045c4c5e46","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/a653f4045c4c5e46","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047885136,"stop":1726047885136,"duration":0}},{"uid":"bfc973c321c1c3b1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/bfc973c321c1c3b1","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037916392,"stop":1726037916392,"duration":0}}]},"c4f3bc323a890a8c1c21c8ac6cff8b21":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":35,"unknown":0,"total":36},"items":[{"uid":"c9648278ab03228c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/c9648278ab03228c","status":"passed","time":{"start":1727169154035,"stop":1727169154035,"duration":0}},{"uid":"8785b7ffbfea2422","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/8785b7ffbfea2422","status":"passed","time":{"start":1727152502654,"stop":1727152502655,"duration":1}},{"uid":"a944df95d04666dd","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/a944df95d04666dd","status":"passed","time":{"start":1727148733268,"stop":1727148733269,"duration":1}},{"uid":"1016b449ca750937","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/1016b449ca750937","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088034316,"stop":1727088034316,"duration":0}},{"uid":"568ad28828b9b109","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/568ad28828b9b109","status":"passed","time":{"start":1726811571780,"stop":1726811571780,"duration":0}},{"uid":"28bd3dbe3fba3471","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/28bd3dbe3fba3471","status":"passed","time":{"start":1726738987918,"stop":1726738987918,"duration":0}},{"uid":"b07a5814944bdea1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/b07a5814944bdea1","status":"passed","time":{"start":1726730682833,"stop":1726730682833,"duration":0}},{"uid":"59d81855308aee3a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/59d81855308aee3a","status":"passed","time":{"start":1726730664300,"stop":1726730664300,"duration":0}},{"uid":"1785735941417187","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/1785735941417187","status":"passed","time":{"start":1726655638662,"stop":1726655638662,"duration":0}},{"uid":"c555305c19eea776","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/c555305c19eea776","status":"passed","time":{"start":1726572394809,"stop":1726572394809,"duration":0}},{"uid":"345ea2c11c02ebfe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/345ea2c11c02ebfe","status":"passed","time":{"start":1726571194127,"stop":1726571194127,"duration":0}},{"uid":"ab4d57917fac3990","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/ab4d57917fac3990","status":"passed","time":{"start":1726555653862,"stop":1726555653862,"duration":0}},{"uid":"6d4926efe87b66af","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/6d4926efe87b66af","status":"passed","time":{"start":1726554684653,"stop":1726554684653,"duration":0}},{"uid":"d356c16296bd05b0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/d356c16296bd05b0","status":"passed","time":{"start":1726552150502,"stop":1726552150503,"duration":1}},{"uid":"7496fdd180240454","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/7496fdd180240454","status":"passed","time":{"start":1726475580730,"stop":1726475580730,"duration":0}},{"uid":"c6cd4c79b60723e3","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/c6cd4c79b60723e3","status":"passed","time":{"start":1726461466501,"stop":1726461466502,"duration":1}},{"uid":"99e646be4678625a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/99e646be4678625a","status":"passed","time":{"start":1726056110528,"stop":1726056110528,"duration":0}},{"uid":"be3b43d06c7e09b1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/be3b43d06c7e09b1","status":"passed","time":{"start":1726051674774,"stop":1726051674774,"duration":0}},{"uid":"d99153a706978977","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/d99153a706978977","status":"passed","time":{"start":1726047884581,"stop":1726047884581,"duration":0}},{"uid":"d10685cc943263f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/d10685cc943263f","status":"passed","time":{"start":1726037915822,"stop":1726037915822,"duration":0}}]},"facd427b63b92a7b83f4ac87ab0c7af0":{"statistic":{"failed":3,"broken":0,"skipped":0,"passed":33,"unknown":0,"total":36},"items":[{"uid":"b815bb7a7480d674","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/b815bb7a7480d674","status":"passed","time":{"start":1727169154156,"stop":1727169154156,"duration":0}},{"uid":"fa8d39998a08c154","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/fa8d39998a08c154","status":"passed","time":{"start":1727152502775,"stop":1727152502775,"duration":0}},{"uid":"8a73ff1c0e5b22b7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/8a73ff1c0e5b22b7","status":"passed","time":{"start":1727148733383,"stop":1727148733383,"duration":0}},{"uid":"b88f09f4c693af57","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/b88f09f4c693af57","status":"passed","time":{"start":1727088034762,"stop":1727088034762,"duration":0}},{"uid":"acfbdcac62533fa7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/acfbdcac62533fa7","status":"failed","statusDetails":"AssertionError: assert False == True\n +  where False = <built-in method get of dict object at 0x7f3bd7220b00>('isSuperUser')\n +    where <built-in method get of dict object at 0x7f3bd7220b00> = {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False}.get\n +      where {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False} = json()\n +        where json = <Response [200]>.json","time":{"start":1726811571901,"stop":1726811571902,"duration":1}},{"uid":"19f56b07a000eec8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/19f56b07a000eec8","status":"failed","statusDetails":"AssertionError: assert False == True\n +  where False = <built-in method get of dict object at 0x7f6285abad00>('isSuperUser')\n +    where <built-in method get of dict object at 0x7f6285abad00> = {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False}.get\n +      where {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False} = json()\n +        where json = <Response [200]>.json","time":{"start":1726738988038,"stop":1726738988038,"duration":0}},{"uid":"ab55995ec7ef65a8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/ab55995ec7ef65a8","status":"failed","statusDetails":"AssertionError: assert False == True\n +  where False = <built-in method get of dict object at 0x7fc6d5a64f40>('isSuperUser')\n +    where <built-in method get of dict object at 0x7fc6d5a64f40> = {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False}.get\n +      where {'address': '1b91f0800ce863abc2b3cc85321d2c8d554ca491767fc68ad5bcf4f3', 'created_at': '2024-09-19 07:24:42.270000+00:00', 'isSuperUser': False} = json()\n +        where json = <Response [200]>.json","time":{"start":1726730683014,"stop":1726730683014,"duration":0}},{"uid":"47aa40a00f447d69","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/47aa40a00f447d69","status":"passed","time":{"start":1726730664421,"stop":1726730664421,"duration":0}},{"uid":"7f0034eaa3e7d77d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/7f0034eaa3e7d77d","status":"passed","time":{"start":1726655638795,"stop":1726655638795,"duration":0}},{"uid":"6d9a51786b314608","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/6d9a51786b314608","status":"passed","time":{"start":1726572394989,"stop":1726572394989,"duration":0}},{"uid":"94845702670c89af","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/94845702670c89af","status":"passed","time":{"start":1726571194260,"stop":1726571194260,"duration":0}},{"uid":"44d34f089e7ea4d5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/44d34f089e7ea4d5","status":"passed","time":{"start":1726555653996,"stop":1726555653996,"duration":0}},{"uid":"42549c1796d6cf4a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/42549c1796d6cf4a","status":"passed","time":{"start":1726554684833,"stop":1726554684833,"duration":0}},{"uid":"561efeb720743920","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/561efeb720743920","status":"passed","time":{"start":1726552150682,"stop":1726552150682,"duration":0}},{"uid":"b09607b25f5a9997","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/b09607b25f5a9997","status":"passed","time":{"start":1726475580910,"stop":1726475580910,"duration":0}},{"uid":"2f9ddba5120c73ee","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/2f9ddba5120c73ee","status":"passed","time":{"start":1726461466622,"stop":1726461466622,"duration":0}},{"uid":"42ea1c953407305a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/42ea1c953407305a","status":"passed","time":{"start":1726056110662,"stop":1726056110662,"duration":0}},{"uid":"a7e63df2768aa2ad","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/a7e63df2768aa2ad","status":"passed","time":{"start":1726051674906,"stop":1726051674906,"duration":0}},{"uid":"d1e83b76df4c9210","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/d1e83b76df4c9210","status":"passed","time":{"start":1726047884714,"stop":1726047884714,"duration":0}},{"uid":"8db32535cf9920b1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/8db32535cf9920b1","status":"passed","time":{"start":1726037915955,"stop":1726037915955,"duration":0}}]},"4614fbf5d189d669b2a9afbe419c31b6":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":35,"unknown":0,"total":36},"items":[{"uid":"4cffdea31fb8a16a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/4cffdea31fb8a16a","status":"passed","time":{"start":1727169154402,"stop":1727169154539,"duration":137}},{"uid":"d0e7afc9024c7948","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d0e7afc9024c7948","status":"passed","time":{"start":1727152503018,"stop":1727152503154,"duration":136}},{"uid":"9ea49321bb29f5be","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/9ea49321bb29f5be","status":"passed","time":{"start":1727148733615,"stop":1727148733746,"duration":131}},{"uid":"a4d6715fef824f0a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/a4d6715fef824f0a","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/templates (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088038511,"stop":1727088038736,"duration":225}},{"uid":"79d9b321fad67c9e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/79d9b321fad67c9e","status":"passed","time":{"start":1726811572060,"stop":1726811572197,"duration":137}},{"uid":"f7462f4be5e9d159","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/f7462f4be5e9d159","status":"passed","time":{"start":1726738988192,"stop":1726738988338,"duration":146}},{"uid":"98d020e9e7c26fd","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/98d020e9e7c26fd","status":"passed","time":{"start":1726730683233,"stop":1726730683442,"duration":209}},{"uid":"b09c42fbc251c013","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/b09c42fbc251c013","status":"passed","time":{"start":1726730664670,"stop":1726730664823,"duration":153}},{"uid":"8f46c46ba261a968","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/8f46c46ba261a968","status":"passed","time":{"start":1726655639068,"stop":1726655639224,"duration":156}},{"uid":"26739bf536a53e84","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/26739bf536a53e84","status":"passed","time":{"start":1726572395356,"stop":1726572395572,"duration":216}},{"uid":"3c3a6932dfd347f0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/3c3a6932dfd347f0","status":"passed","time":{"start":1726571194535,"stop":1726571194707,"duration":172}},{"uid":"76a2f1f9d9ae9140","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/76a2f1f9d9ae9140","status":"passed","time":{"start":1726555654273,"stop":1726555654438,"duration":165}},{"uid":"e6fad4188395b8b8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/e6fad4188395b8b8","status":"passed","time":{"start":1726554685199,"stop":1726554685406,"duration":207}},{"uid":"16b08014ec6b57be","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/16b08014ec6b57be","status":"passed","time":{"start":1726552151046,"stop":1726552151257,"duration":211}},{"uid":"897c536ab32bfe60","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/897c536ab32bfe60","status":"passed","time":{"start":1726475581277,"stop":1726475581487,"duration":210}},{"uid":"84a8495bf0bcdd28","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/84a8495bf0bcdd28","status":"passed","time":{"start":1726461466866,"stop":1726461467020,"duration":154}},{"uid":"82b3a52f9139dd78","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/82b3a52f9139dd78","status":"passed","time":{"start":1726056110933,"stop":1726056111098,"duration":165}},{"uid":"dc6ffe5d153ac3f5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/dc6ffe5d153ac3f5","status":"passed","time":{"start":1726051675175,"stop":1726051675329,"duration":154}},{"uid":"7f21a6acd662e0fe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/7f21a6acd662e0fe","status":"passed","time":{"start":1726047884986,"stop":1726047885134,"duration":148}},{"uid":"b4d74127f4d14957","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/b4d74127f4d14957","status":"passed","time":{"start":1726037916227,"stop":1726037916390,"duration":163}}]},"b479afaeccaeb0b2b306544ea95f6f33":{"statistic":{"failed":23,"broken":0,"skipped":0,"passed":13,"unknown":0,"total":36},"items":[{"uid":"5185c32d22a28357","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/5185c32d22a28357","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1727169153682,"stop":1727169153906,"duration":224}},{"uid":"e1a585f972b501d6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/e1a585f972b501d6","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1727152502322,"stop":1727152502527,"duration":205}},{"uid":"7d4044701f90a0d7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/7d4044701f90a0d7","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1727148732943,"stop":1727148733146,"duration":203}},{"uid":"862a4a22da1537fc","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/862a4a22da1537fc","status":"failed","statusDetails":"AssertionError: Expected GET /trigger-history to succeed but got statusCode: 500 : body: <html>\r\n<head><title>500 Internal Server Error</title></head>\r\n<body>\r\n<center><h1>500 Internal Server Error</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1727088033942,"stop":1727088034307,"duration":365}},{"uid":"e64c174a14c6d1d2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/e64c174a14c6d1d2","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571643,"stop":1726811571643,"duration":0}},{"uid":"c2237c532f28a240","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/c2237c532f28a240","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987781,"stop":1726738987781,"duration":0}},{"uid":"6e0f95324875fb29","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/6e0f95324875fb29","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682634,"stop":1726730682634,"duration":0}},{"uid":"63b8dc8337513f55","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/63b8dc8337513f55","status":"passed","time":{"start":1726730663932,"stop":1726730664165,"duration":233}},{"uid":"f0b812c32042d35e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/f0b812c32042d35e","status":"passed","time":{"start":1726655638283,"stop":1726655638523,"duration":240}},{"uid":"7d65815907a040d8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/7d65815907a040d8","status":"passed","time":{"start":1726572394314,"stop":1726572394623,"duration":309}},{"uid":"8f7444fed641fae4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/8f7444fed641fae4","status":"passed","time":{"start":1726571193681,"stop":1726571193915,"duration":234}},{"uid":"cc132a97b9af12a6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/cc132a97b9af12a6","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1726555653466,"stop":1726555653719,"duration":253}},{"uid":"1c0ff00be3bd4159","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/1c0ff00be3bd4159","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1726554684173,"stop":1726554684465,"duration":292}},{"uid":"cb21bad41af78b8e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/cb21bad41af78b8e","status":"passed","time":{"start":1726552150044,"stop":1726552150319,"duration":275}},{"uid":"f7c13d94199f8af1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/f7c13d94199f8af1","status":"passed","time":{"start":1726475580238,"stop":1726475580544,"duration":306}},{"uid":"2385d0f1bb4cf79f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/2385d0f1bb4cf79f","status":"failed","statusDetails":"AssertionError: No successful 'voteEvent' log found | Failed Logs: None","time":{"start":1726461466088,"stop":1726461466345,"duration":257}},{"uid":"803db4dbaf8e5d20","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/803db4dbaf8e5d20","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110368,"stop":1726056110368,"duration":0}},{"uid":"222a24e96582bdef","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/222a24e96582bdef","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674615,"stop":1726051674615,"duration":0}},{"uid":"e29cc00eda7bea2a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/e29cc00eda7bea2a","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884416,"stop":1726047884416,"duration":0}},{"uid":"7af6a59f2c499a68","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/7af6a59f2c499a68","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915661,"stop":1726037915661,"duration":0}}]},"d79a2535db45bda6b7c50f297d55cad0":{"statistic":{"failed":3,"broken":0,"skipped":0,"passed":33,"unknown":0,"total":36},"items":[{"uid":"aac57750900fd4ec","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/aac57750900fd4ec","status":"passed","time":{"start":1727169154540,"stop":1727169154540,"duration":0}},{"uid":"772a0c88a4cd5209","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/772a0c88a4cd5209","status":"passed","time":{"start":1727152503155,"stop":1727152503155,"duration":0}},{"uid":"eb770d38c9b9f37a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/eb770d38c9b9f37a","status":"passed","time":{"start":1727148733748,"stop":1727148733748,"duration":0}},{"uid":"19bcc49628c976c7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/19bcc49628c976c7","status":"passed","time":{"start":1727088038882,"stop":1727088038883,"duration":1}},{"uid":"9b44297d711d52b9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/9b44297d711d52b9","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811572197,"stop":1726811572197,"duration":0}},{"uid":"dfdbc3c44dfba3f6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/dfdbc3c44dfba3f6","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738988339,"stop":1726738988339,"duration":0}},{"uid":"26974eb38f9eced1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/26974eb38f9eced1","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730683443,"stop":1726730683443,"duration":0}},{"uid":"9054698fc351c171","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/9054698fc351c171","status":"passed","time":{"start":1726730664824,"stop":1726730664824,"duration":0}},{"uid":"34960c8db77799ae","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/34960c8db77799ae","status":"passed","time":{"start":1726655639225,"stop":1726655639226,"duration":1}},{"uid":"6ccd2bab97193ecf","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/6ccd2bab97193ecf","status":"passed","time":{"start":1726572395574,"stop":1726572395574,"duration":0}},{"uid":"b26d5db71b5eead8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/b26d5db71b5eead8","status":"passed","time":{"start":1726571194708,"stop":1726571194708,"duration":0}},{"uid":"82fea142705e82ce","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/82fea142705e82ce","status":"passed","time":{"start":1726555654439,"stop":1726555654440,"duration":1}},{"uid":"d0c3191963225043","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/d0c3191963225043","status":"passed","time":{"start":1726554685407,"stop":1726554685407,"duration":0}},{"uid":"4ef9268b56f322f2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/4ef9268b56f322f2","status":"passed","time":{"start":1726552151258,"stop":1726552151258,"duration":0}},{"uid":"827416a1d0cf6806","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/827416a1d0cf6806","status":"passed","time":{"start":1726475581489,"stop":1726475581489,"duration":0}},{"uid":"28734ffffdf3ce52","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/28734ffffdf3ce52","status":"passed","time":{"start":1726461467021,"stop":1726461467021,"duration":0}},{"uid":"d53171d64e4440a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/d53171d64e4440a","status":"passed","time":{"start":1726056111099,"stop":1726056111099,"duration":0}},{"uid":"11356dff2ea898e1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/11356dff2ea898e1","status":"passed","time":{"start":1726051675330,"stop":1726051675330,"duration":0}},{"uid":"4f2af32d2b6094aa","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/4f2af32d2b6094aa","status":"passed","time":{"start":1726047885135,"stop":1726047885135,"duration":0}},{"uid":"b9d6a5fafa4752d7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/b9d6a5fafa4752d7","status":"passed","time":{"start":1726037916391,"stop":1726037916391,"duration":0}}]},"3e0654fc84f60fdcfbe40b395f79c630":{"statistic":{"failed":3,"broken":1,"skipped":0,"passed":32,"unknown":0,"total":36},"items":[{"uid":"7a45cec27bd29330","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/7a45cec27bd29330","status":"passed","time":{"start":1727169154278,"stop":1727169154401,"duration":123}},{"uid":"b6d74a76c8090dd2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/b6d74a76c8090dd2","status":"passed","time":{"start":1727152502896,"stop":1727152503017,"duration":121}},{"uid":"5bf29bb422c1a53c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/5bf29bb422c1a53c","status":"passed","time":{"start":1727148733498,"stop":1727148733614,"duration":116}},{"uid":"b99ae9f246aba9b1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/b99ae9f246aba9b1","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/templates/a877a86c-2834-4d44-9f4c-6a5a9398be26 (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088038179,"stop":1727088038405,"duration":226}},{"uid":"161a289ebe9947e2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/161a289ebe9947e2","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811572049,"stop":1726811572049,"duration":0}},{"uid":"ff4b781c104189d0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/ff4b781c104189d0","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738988181,"stop":1726738988181,"duration":0}},{"uid":"cfcc41d25c1ffc10","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/cfcc41d25c1ffc10","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730683222,"stop":1726730683222,"duration":0}},{"uid":"20048666921dab21","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/20048666921dab21","status":"passed","time":{"start":1726730664542,"stop":1726730664669,"duration":127}},{"uid":"78640264fcc3f32","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/78640264fcc3f32","status":"passed","time":{"start":1726655638929,"stop":1726655639067,"duration":138}},{"uid":"22b62fbdae3d2d17","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/22b62fbdae3d2d17","status":"passed","time":{"start":1726572395170,"stop":1726572395354,"duration":184}},{"uid":"ff3a0cd228ce96b3","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/ff3a0cd228ce96b3","status":"passed","time":{"start":1726571194394,"stop":1726571194534,"duration":140}},{"uid":"f3e7ae0f88d7fa36","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/f3e7ae0f88d7fa36","status":"passed","time":{"start":1726555654130,"stop":1726555654272,"duration":142}},{"uid":"fdf2dfffb947e87f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/fdf2dfffb947e87f","status":"passed","time":{"start":1726554685014,"stop":1726554685198,"duration":184}},{"uid":"a2fbf8d68875d15c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/a2fbf8d68875d15c","status":"passed","time":{"start":1726552150862,"stop":1726552151045,"duration":183}},{"uid":"4f8b488d59be0f1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/4f8b488d59be0f1","status":"passed","time":{"start":1726475581091,"stop":1726475581276,"duration":185}},{"uid":"2d0e38f2d044df24","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/2d0e38f2d044df24","status":"passed","time":{"start":1726461466744,"stop":1726461466865,"duration":121}},{"uid":"69d25d5ad4a79805","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/69d25d5ad4a79805","status":"passed","time":{"start":1726056110798,"stop":1726056110932,"duration":134}},{"uid":"e2b34877c275dfed","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/e2b34877c275dfed","status":"passed","time":{"start":1726051675040,"stop":1726051675174,"duration":134}},{"uid":"1967e002e1c3ab66","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/1967e002e1c3ab66","status":"passed","time":{"start":1726047884848,"stop":1726047884985,"duration":137}},{"uid":"42675c011a65d4a4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/42675c011a65d4a4","status":"passed","time":{"start":1726037916090,"stop":1726037916226,"duration":136}}]},"06580cb02b339767857019902ae34541":{"statistic":{"failed":3,"broken":0,"skipped":0,"passed":33,"unknown":0,"total":36},"items":[{"uid":"416e85821066bb49","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/416e85821066bb49","status":"passed","time":{"start":1727168667844,"stop":1727168668283,"duration":439}},{"uid":"31fdc3fbc28d00ed","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/31fdc3fbc28d00ed","status":"passed","time":{"start":1727152015717,"stop":1727152015848,"duration":131}},{"uid":"267cd2d556266f68","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/267cd2d556266f68","status":"passed","time":{"start":1727148246526,"stop":1727148246863,"duration":337}},{"uid":"3c64a7f4f0bdb638","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/3c64a7f4f0bdb638","status":"passed","time":{"start":1727087529256,"stop":1727087529568,"duration":312}},{"uid":"8fbd096da728fdc7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/8fbd096da728fdc7","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571302,"stop":1726811571302,"duration":0}},{"uid":"f7da25bd7c82f732","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/f7da25bd7c82f732","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987446,"stop":1726738987446,"duration":0}},{"uid":"44a82aa27eec6351","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/44a82aa27eec6351","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682178,"stop":1726730682178,"duration":0}},{"uid":"698a7fa34fe547f2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/698a7fa34fe547f2","status":"passed","time":{"start":1726730178615,"stop":1726730178938,"duration":323}},{"uid":"19732cbe9cad6d5f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/19732cbe9cad6d5f","status":"passed","time":{"start":1726655152991,"stop":1726655153327,"duration":336}},{"uid":"fca9c0f550a362eb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/fca9c0f550a362eb","status":"passed","time":{"start":1726571908367,"stop":1726571908736,"duration":369}},{"uid":"fe43614aa904265c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/fe43614aa904265c","status":"passed","time":{"start":1726570708256,"stop":1726570708607,"duration":351}},{"uid":"d05bbaf56fbb1afe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/d05bbaf56fbb1afe","status":"passed","time":{"start":1726555168195,"stop":1726555168532,"duration":337}},{"uid":"cf7096139a49c034","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/cf7096139a49c034","status":"passed","time":{"start":1726554197488,"stop":1726554197884,"duration":396}},{"uid":"f2319149493f06fc","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/f2319149493f06fc","status":"passed","time":{"start":1726551663856,"stop":1726551664249,"duration":393}},{"uid":"8adbb17bd30fb965","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/8adbb17bd30fb965","status":"passed","time":{"start":1726475093641,"stop":1726475094058,"duration":417}},{"uid":"a705e5399bed053a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/a705e5399bed053a","status":"passed","time":{"start":1726460969569,"stop":1726460969900,"duration":331}},{"uid":"a5b8285c1198b538","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/a5b8285c1198b538","status":"passed","time":{"start":1726056109810,"stop":1726056110128,"duration":318}},{"uid":"e046da67ab130e32","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/e046da67ab130e32","status":"passed","time":{"start":1726051674039,"stop":1726051674383,"duration":344}},{"uid":"dffa4566fcda1648","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/dffa4566fcda1648","status":"passed","time":{"start":1726047883777,"stop":1726047884151,"duration":374}},{"uid":"10f29bfa212887d8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/10f29bfa212887d8","status":"passed","time":{"start":1726037915051,"stop":1726037915416,"duration":365}}]},"f3a838b08baad92152078eb3339eeab2":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":35,"unknown":0,"total":36},"items":[{"uid":"31e55d45be6c221b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/31e55d45be6c221b","status":"passed","time":{"start":1727169154157,"stop":1727169154275,"duration":118}},{"uid":"8cfcd4cc70bab607","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/8cfcd4cc70bab607","status":"passed","time":{"start":1727152502776,"stop":1727152502893,"duration":117}},{"uid":"205fc5031363f4ec","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/205fc5031363f4ec","status":"passed","time":{"start":1727148733384,"stop":1727148733495,"duration":111}},{"uid":"ecd98ce37f8d81c2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/ecd98ce37f8d81c2","status":"failed","statusDetails":"AssertionError: Expected POST /auth/logout to succeed but got statusCode: 502 : body: <html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1727088034763,"stop":1727088038168,"duration":3405}},{"uid":"e41b86c79f226b51","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/e41b86c79f226b51","status":"passed","time":{"start":1726811571905,"stop":1726811572024,"duration":119}},{"uid":"24b6f5afdfdbd0d0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/24b6f5afdfdbd0d0","status":"passed","time":{"start":1726738988041,"stop":1726738988159,"duration":118}},{"uid":"ca638759a1c7b9c5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/ca638759a1c7b9c5","status":"passed","time":{"start":1726730683017,"stop":1726730683195,"duration":178}},{"uid":"d7d67c9e9ad6ff87","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/d7d67c9e9ad6ff87","status":"passed","time":{"start":1726730664422,"stop":1726730664539,"duration":117}},{"uid":"1db0fa78741b1e6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/1db0fa78741b1e6","status":"passed","time":{"start":1726655638796,"stop":1726655638926,"duration":130}},{"uid":"12112eaadf9dffef","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/12112eaadf9dffef","status":"passed","time":{"start":1726572394990,"stop":1726572395167,"duration":177}},{"uid":"1ca961ecf472d9c9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/1ca961ecf472d9c9","status":"passed","time":{"start":1726571194261,"stop":1726571194391,"duration":130}},{"uid":"2106eb6a2c98a75","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/2106eb6a2c98a75","status":"passed","time":{"start":1726555653997,"stop":1726555654127,"duration":130}},{"uid":"21443561de5bd179","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/21443561de5bd179","status":"passed","time":{"start":1726554684834,"stop":1726554685011,"duration":177}},{"uid":"3758b0cf514d69fb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/3758b0cf514d69fb","status":"passed","time":{"start":1726552150683,"stop":1726552150860,"duration":177}},{"uid":"a37df65fcf23b32e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/a37df65fcf23b32e","status":"passed","time":{"start":1726475580911,"stop":1726475581088,"duration":177}},{"uid":"36d2ea16f105b3ff","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/36d2ea16f105b3ff","status":"passed","time":{"start":1726461466623,"stop":1726461466741,"duration":118}},{"uid":"8717929c63fe2295","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/8717929c63fe2295","status":"passed","time":{"start":1726056110663,"stop":1726056110795,"duration":132}},{"uid":"ad91fea07485d3e6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/ad91fea07485d3e6","status":"passed","time":{"start":1726051674907,"stop":1726051675037,"duration":130}},{"uid":"1931e5fb95fcd23f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/1931e5fb95fcd23f","status":"passed","time":{"start":1726047884715,"stop":1726047884845,"duration":130}},{"uid":"ea35eeed158760b4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/ea35eeed158760b4","status":"passed","time":{"start":1726037915956,"stop":1726037916087,"duration":131}}]},"e60a8efb6389f125260a50658880d0d3":{"statistic":{"failed":3,"broken":0,"skipped":0,"passed":33,"unknown":0,"total":36},"items":[{"uid":"b08d637de595b7ac","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/b08d637de595b7ac","status":"passed","time":{"start":1727169154277,"stop":1727169154277,"duration":0}},{"uid":"d2a0a2de11b54ca2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d2a0a2de11b54ca2","status":"passed","time":{"start":1727152502895,"stop":1727152502895,"duration":0}},{"uid":"d340729ae7280e1f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d340729ae7280e1f","status":"passed","time":{"start":1727148733497,"stop":1727148733497,"duration":0}},{"uid":"43a280ef989748cd","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/43a280ef989748cd","status":"passed","time":{"start":1727088038178,"stop":1727088038178,"duration":0}},{"uid":"dd60dd9780f847de","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/dd60dd9780f847de","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811572025,"stop":1726811572025,"duration":0}},{"uid":"5b53f057b42af449","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/5b53f057b42af449","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738988160,"stop":1726738988160,"duration":0}},{"uid":"107edb0e503773e4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/107edb0e503773e4","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730683196,"stop":1726730683196,"duration":0}},{"uid":"acede39165b8b832","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/acede39165b8b832","status":"passed","time":{"start":1726730664541,"stop":1726730664541,"duration":0}},{"uid":"c37efb6cc093c0f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/c37efb6cc093c0f","status":"passed","time":{"start":1726655638928,"stop":1726655638928,"duration":0}},{"uid":"891cb3a94ea9a67f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/891cb3a94ea9a67f","status":"passed","time":{"start":1726572395169,"stop":1726572395169,"duration":0}},{"uid":"d81649b9c0e7ebc","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/d81649b9c0e7ebc","status":"passed","time":{"start":1726571194393,"stop":1726571194393,"duration":0}},{"uid":"cebe71a003d36719","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/cebe71a003d36719","status":"passed","time":{"start":1726555654129,"stop":1726555654129,"duration":0}},{"uid":"268105abe2e8fa7c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/268105abe2e8fa7c","status":"passed","time":{"start":1726554685012,"stop":1726554685013,"duration":1}},{"uid":"164151396af686ac","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/164151396af686ac","status":"passed","time":{"start":1726552150861,"stop":1726552150861,"duration":0}},{"uid":"9ffef511ad2386f0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/9ffef511ad2386f0","status":"passed","time":{"start":1726475581090,"stop":1726475581090,"duration":0}},{"uid":"a50cbe1f5d8f304c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/a50cbe1f5d8f304c","status":"passed","time":{"start":1726461466743,"stop":1726461466743,"duration":0}},{"uid":"87c492e9e7f3c11e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/87c492e9e7f3c11e","status":"passed","time":{"start":1726056110797,"stop":1726056110797,"duration":0}},{"uid":"8e01ffbe3fa94414","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/8e01ffbe3fa94414","status":"passed","time":{"start":1726051675039,"stop":1726051675039,"duration":0}},{"uid":"fd55a53d392ebe3c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/fd55a53d392ebe3c","status":"passed","time":{"start":1726047884847,"stop":1726047884847,"duration":0}},{"uid":"73071510b9a6cd4e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/73071510b9a6cd4e","status":"passed","time":{"start":1726037916089,"stop":1726037916089,"duration":0}}]},"398b8ccba1d43478bb443a9fb65225bb":{"statistic":{"failed":9,"broken":0,"skipped":0,"passed":27,"unknown":0,"total":36},"items":[{"uid":"9dc3b10cdeb18210","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/9dc3b10cdeb18210","status":"passed","time":{"start":1727168668437,"stop":1727168668437,"duration":0}},{"uid":"776b5095722280c2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/776b5095722280c2","status":"passed","time":{"start":1727152015996,"stop":1727152015996,"duration":0}},{"uid":"cfce5326e89de474","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/cfce5326e89de474","status":"passed","time":{"start":1727148247007,"stop":1727148247008,"duration":1}},{"uid":"4ebb9b5bb0e00a10","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/4ebb9b5bb0e00a10","status":"passed","time":{"start":1727087529710,"stop":1727087529710,"duration":0}},{"uid":"ea1ea465585d51ff","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/ea1ea465585d51ff","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571597,"stop":1726811571597,"duration":0}},{"uid":"7b5a4b0d77194931","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/7b5a4b0d77194931","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987736,"stop":1726738987736,"duration":0}},{"uid":"9752ebbf1f79ed12","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/9752ebbf1f79ed12","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682588,"stop":1726730682588,"duration":0}},{"uid":"cb35ec35359f2c5f","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/cb35ec35359f2c5f","status":"passed","time":{"start":1726730179080,"stop":1726730179081,"duration":1}},{"uid":"f1173806f1a79e1a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/f1173806f1a79e1a","status":"passed","time":{"start":1726655153489,"stop":1726655153489,"duration":0}},{"uid":"53e509da9fbc8b01","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/53e509da9fbc8b01","status":"passed","time":{"start":1726571908952,"stop":1726571908952,"duration":0}},{"uid":"31aac9d305611143","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/31aac9d305611143","status":"passed","time":{"start":1726570708773,"stop":1726570708774,"duration":1}},{"uid":"19696f41a0efea67","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/19696f41a0efea67","status":"passed","time":{"start":1726555168682,"stop":1726555168683,"duration":1}},{"uid":"89334da42a4c17a3","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/89334da42a4c17a3","status":"passed","time":{"start":1726554198090,"stop":1726554198090,"duration":0}},{"uid":"bc9fdfa5e7da4198","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/bc9fdfa5e7da4198","status":"passed","time":{"start":1726551664463,"stop":1726551664463,"duration":0}},{"uid":"b1da6c3ce7425df9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/b1da6c3ce7425df9","status":"passed","time":{"start":1726475094270,"stop":1726475094271,"duration":1}},{"uid":"347825b31083fd21","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/347825b31083fd21","status":"passed","time":{"start":1726460970054,"stop":1726460970054,"duration":0}},{"uid":"fea0fc9047834e3e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/fea0fc9047834e3e","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110129,"stop":1726056110129,"duration":0}},{"uid":"1890a152cd512a10","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/1890a152cd512a10","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674383,"stop":1726051674383,"duration":0}},{"uid":"7e7083638143b571","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/7e7083638143b571","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884152,"stop":1726047884152,"duration":0}},{"uid":"89298c3828d291ed","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/89298c3828d291ed","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915417,"stop":1726037915418,"duration":1}}]},"bf5102b0fa26be747eb8ee000c1d24e7":{"statistic":{"failed":10,"broken":0,"skipped":0,"passed":26,"unknown":0,"total":36},"items":[{"uid":"21377587135cbd38","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/21377587135cbd38","status":"passed","time":{"start":1727169153190,"stop":1727169153190,"duration":0}},{"uid":"67318868f08274d8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/67318868f08274d8","status":"passed","time":{"start":1727152501864,"stop":1727152501864,"duration":0}},{"uid":"8b78a99ed7a7628e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/8b78a99ed7a7628e","status":"passed","time":{"start":1727148732498,"stop":1727148732498,"duration":0}},{"uid":"d1b1ff7668a192c4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d1b1ff7668a192c4","status":"failed","statusDetails":"AssertionError: Expected DELETE /agents/cbccb0ca-8534-47ca-95e0-0020248e9820 to succeed but got statusCode: 502 : body: <html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1727087529711,"stop":1727087529711,"duration":0}},{"uid":"9a4807b546197b6c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/9a4807b546197b6c","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571608,"stop":1726811571608,"duration":0}},{"uid":"7ad5abed68ee7076","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/7ad5abed68ee7076","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987747,"stop":1726738987747,"duration":0}},{"uid":"1668de7329c98266","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/1668de7329c98266","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682599,"stop":1726730682599,"duration":0}},{"uid":"ec882a94aecb03fe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/ec882a94aecb03fe","status":"passed","time":{"start":1726730663450,"stop":1726730663450,"duration":0}},{"uid":"426b1badecc9458b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/426b1badecc9458b","status":"passed","time":{"start":1726655637826,"stop":1726655637827,"duration":1}},{"uid":"3ed0b4844decbb54","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/3ed0b4844decbb54","status":"passed","time":{"start":1726572393689,"stop":1726572393689,"duration":0}},{"uid":"544c4eb34eafa9d6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/544c4eb34eafa9d6","status":"passed","time":{"start":1726571193182,"stop":1726571193182,"duration":0}},{"uid":"4d44cd4ed3144eea","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/4d44cd4ed3144eea","status":"passed","time":{"start":1726555652878,"stop":1726555652878,"duration":0}},{"uid":"b2c41dfbaf75dd00","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/b2c41dfbaf75dd00","status":"passed","time":{"start":1726554683557,"stop":1726554683557,"duration":0}},{"uid":"5527423b23ff0a5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/5527423b23ff0a5","status":"passed","time":{"start":1726552149495,"stop":1726552149495,"duration":0}},{"uid":"f09407a36b3eee27","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/f09407a36b3eee27","status":"passed","time":{"start":1726475579615,"stop":1726475579616,"duration":1}},{"uid":"e7fc9329dd2d9ed5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/e7fc9329dd2d9ed5","status":"passed","time":{"start":1726461465537,"stop":1726461465537,"duration":0}},{"uid":"627e4c8c897ce279","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/627e4c8c897ce279","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110333,"stop":1726056110333,"duration":0}},{"uid":"28cf64adad376add","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/28cf64adad376add","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674581,"stop":1726051674581,"duration":0}},{"uid":"1f880bfce7aeabef","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/1f880bfce7aeabef","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884375,"stop":1726047884375,"duration":0}},{"uid":"a3fadd300fb153c1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/a3fadd300fb153c1","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915625,"stop":1726037915625,"duration":0}}]},"8f66ed3039064d88cfc2340a1bec5701":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":35,"unknown":0,"total":36},"items":[{"uid":"f3811e7c50ac32a8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/f3811e7c50ac32a8","status":"passed","time":{"start":1727169154036,"stop":1727169154154,"duration":118}},{"uid":"2628d511503f7f4d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/2628d511503f7f4d","status":"passed","time":{"start":1727152502656,"stop":1727152502774,"duration":118}},{"uid":"ba0b834df9fdb230","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/ba0b834df9fdb230","status":"passed","time":{"start":1727148733270,"stop":1727148733381,"duration":111}},{"uid":"e49f8f0eafd22861","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/e49f8f0eafd22861","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))","time":{"start":1727088034645,"stop":1727088034645,"duration":0}},{"uid":"df29ea72debf00fc","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/df29ea72debf00fc","status":"passed","time":{"start":1726811571782,"stop":1726811571900,"duration":118}},{"uid":"1349a4aee94231b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/1349a4aee94231b","status":"passed","time":{"start":1726738987919,"stop":1726738988037,"duration":118}},{"uid":"33f0ec8fd34451c3","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/33f0ec8fd34451c3","status":"passed","time":{"start":1726730682834,"stop":1726730683012,"duration":178}},{"uid":"b85497511cb1d4b9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/b85497511cb1d4b9","status":"passed","time":{"start":1726730664301,"stop":1726730664419,"duration":118}},{"uid":"2d4090cbf5258a91","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/2d4090cbf5258a91","status":"passed","time":{"start":1726655638663,"stop":1726655638794,"duration":131}},{"uid":"2120814ce92df02e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/2120814ce92df02e","status":"passed","time":{"start":1726572394810,"stop":1726572394988,"duration":178}},{"uid":"8ade49893c337534","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/8ade49893c337534","status":"passed","time":{"start":1726571194129,"stop":1726571194258,"duration":129}},{"uid":"77eacf7f2015e4a5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/77eacf7f2015e4a5","status":"passed","time":{"start":1726555653864,"stop":1726555653994,"duration":130}},{"uid":"b54133c4132c2115","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/b54133c4132c2115","status":"passed","time":{"start":1726554684654,"stop":1726554684832,"duration":178}},{"uid":"27de45f6d5f03a17","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/27de45f6d5f03a17","status":"passed","time":{"start":1726552150504,"stop":1726552150681,"duration":177}},{"uid":"2295b404c335b198","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/2295b404c335b198","status":"passed","time":{"start":1726475580731,"stop":1726475580909,"duration":178}},{"uid":"37cf3360775a363","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/37cf3360775a363","status":"passed","time":{"start":1726461466503,"stop":1726461466620,"duration":117}},{"uid":"1be4640478c0dd45","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/1be4640478c0dd45","status":"passed","time":{"start":1726056110530,"stop":1726056110660,"duration":130}},{"uid":"78af5f65c944c37d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/78af5f65c944c37d","status":"passed","time":{"start":1726051674775,"stop":1726051674905,"duration":130}},{"uid":"e769e24a5ab7e731","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/e769e24a5ab7e731","status":"passed","time":{"start":1726047884582,"stop":1726047884712,"duration":130}},{"uid":"2b511e1dbbffb9f7","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/2b511e1dbbffb9f7","status":"passed","time":{"start":1726037915823,"stop":1726037915954,"duration":131}}]},"765acc991ec56447f966e3885e76d93c":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":36,"unknown":0,"total":36},"items":[{"uid":"6759245dc1d0e082","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/35//#testresult/6759245dc1d0e082","status":"passed","time":{"start":1727168666650,"stop":1727168667311,"duration":661}},{"uid":"9b788a88627a6c89","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/9b788a88627a6c89","status":"passed","time":{"start":1727152014607,"stop":1727152015193,"duration":586}},{"uid":"cb12e6407cca1695","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/cb12e6407cca1695","status":"passed","time":{"start":1727148245442,"stop":1727148246009,"duration":567}},{"uid":"556aac7e38cd089d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/556aac7e38cd089d","status":"passed","time":{"start":1727087528219,"stop":1727087528735,"duration":516}},{"uid":"5e98a863cdd966a1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/5e98a863cdd966a1","status":"passed","time":{"start":1726811570695,"stop":1726811571301,"duration":606}},{"uid":"99176519135f9e09","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/99176519135f9e09","status":"passed","time":{"start":1726738986839,"stop":1726738987445,"duration":606}},{"uid":"71427ad901e63727","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/71427ad901e63727","status":"passed","time":{"start":1726730681484,"stop":1726730682177,"duration":693}},{"uid":"6b57d6f33c9497cf","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/6b57d6f33c9497cf","status":"passed","time":{"start":1726730177486,"stop":1726730178086,"duration":600}},{"uid":"a597a096d236e725","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/a597a096d236e725","status":"passed","time":{"start":1726655151809,"stop":1726655152380,"duration":571}},{"uid":"80179fbcc3ccbfb9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/80179fbcc3ccbfb9","status":"passed","time":{"start":1726571906900,"stop":1726571907579,"duration":679}},{"uid":"c37ee7b8004ee51","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/c37ee7b8004ee51","status":"passed","time":{"start":1726570707047,"stop":1726570707632,"duration":585}},{"uid":"9ac7b40ff7cd27b0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/9ac7b40ff7cd27b0","status":"passed","time":{"start":1726555167020,"stop":1726555167625,"duration":605}},{"uid":"aa2860ff2b33bd9c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/aa2860ff2b33bd9c","status":"passed","time":{"start":1726554196120,"stop":1726554196721,"duration":601}},{"uid":"89d265ecb1e3240c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/89d265ecb1e3240c","status":"passed","time":{"start":1726551662343,"stop":1726551663066,"duration":723}},{"uid":"8d49b9a2121e779a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/8d49b9a2121e779a","status":"passed","time":{"start":1726475092087,"stop":1726475092838,"duration":751}},{"uid":"daaee408754ecf6a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/daaee408754ecf6a","status":"passed","time":{"start":1726460968429,"stop":1726460969026,"duration":597}},{"uid":"9d77dccb580b8c41","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/9d77dccb580b8c41","status":"passed","time":{"start":1726056108605,"stop":1726056109220,"duration":615}},{"uid":"6563f143379e09db","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/6563f143379e09db","status":"passed","time":{"start":1726051672807,"stop":1726051673428,"duration":621}},{"uid":"d9c256b33938183d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/d9c256b33938183d","status":"passed","time":{"start":1726047882559,"stop":1726047883183,"duration":624}},{"uid":"4b1368b7e3ba64e0","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/4b1368b7e3ba64e0","status":"passed","time":{"start":1726037913793,"stop":1726037914467,"duration":674}}]}}